exec-ui/src/app/modules/metrics/modules/metrics/performance-test/services/performance-test.service.ts

```diff
@@ -0,0 +1,54 @@
+import {MetricDetail} from '../../../shared/domain-models/metric-detail';
```

Comment:
I do this too but we need to start defining imports consistently. With angular specific imports at the top and project specific imports below that. Can you make sure we are doing this on all the UI files?

Reply:
Import statements reordered in all files.